unittests/fixtures/factory/work-dir/entry_el7_osg34/submit_attrs.cfg

```diff
@@ -0,0 +1,4 @@
+# File: submit_attrs.cfg
+#
+RequestMemory   '2000'
+MySubmitAttr2       'Test12234567i'
```

Comment:
the name used is “MySubmitAttr2". I thought it would be “MySubmitAttr”, is that the case? Or can it be named anything?

Reply:
It is MySubmitAttr2, it is just a fake attribute name for the test